containers/kubernetes/how-to/use-scratch-storage-h100.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: How to use scratch storage with H100 GPUs and Kubernetes Kapsule
+  description: This page explains how to use scratch storage with H100 GPUs and Kubernetes Kapsule
+content:
+  h1: How to use scratch storage with H100 GPUs and Kubernetes Kapsule
+  paragraph: This page explains how to use scratch storage with H100 GPUs and Kubernetes Kapsule
+tags: kubernetes kubernetes-kapsule kapsule cluster gpu-operator nvidia gpu scratch h100
+dates:
+  validation: 2023-12-27
+  posted: 2023-12-27
+categories:
+  - containers
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule or Kosmos cluster
+  - Your cluster is using [H100 GPU Instances](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+</Message>
+
+[Scratch storage](/compute/gpu/concepts/#scratch-storage) is ephemeral storage allocated to H100 GPU Instances (up to 6 TB) and crucial for handling temporary data in high-performance computing environments, especially when working with GPUs like the H100. Kubernetes, a powerful container orchestration platform, can be configured to make optimal use of Scratch storage for improved performance.
+
+Its utility spans multiple use cases, including but not limited to:
+
+- **Large dataset storage for training:** H100 scratch storage provides a local, high-performance storage option for storing large datasets required for training machine learning models. This can significantly enhance the efficiency of training workflows by reducing data transfer times.
+- **Checkpointing during training:** The scratch volume allows for the seamless saving of checkpoints during model training. This is important for resuming training processes in case of interruptions or for exploring different model iterations without starting from the beginning.
+- **Temporary storage for preprocessing:** For data preprocessing tasks that demand high I/O throughput and low latency, H100 scratch storage offers a temporary space on the host node's filesystem. This can be advatageous for quick and efficient preprocessing steps in complex data pipelines.
```

Comment:
```suggestion
- **Temporary storage for preprocessing:** For data preprocessing tasks that demand high I/O throughput and low latency, H100 scratch storage offers a temporary space on the host node's filesystem. This can be advantageous for quick and efficient preprocessing steps in complex data pipelines.
```